src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -62,16 +68,23 @@
 public class SkroutzGreekStemmer {
   public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
   private final CharArraySet stopwords;
+  private Map<String,char[]> step_0_exceptions;
 
   public SkroutzGreekStemmer(final CharArraySet stopwords) {
     this.stopwords = stopwords;
+    this.step_0_exceptions = new HashMap(0);
   }
 
   public SkroutzGreekStemmer() {
     this.stopwords = SkroutzGreekStemmer.getDefaultStopSet();
+    try {
+      this.step_0_exceptions = loadExceptions("step_0_exceptions");
+    } catch(Exception ex) {
+      this.step_0_exceptions =  new HashMap(0);
```

Comment:
It should be better to initialize it at its declaration and avoid initialization in the catch block